main.go

```diff
@@ -30,8 +44,17 @@ func main() {
 	}
 
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		w.WriteHeader(*code)
+
+		for _, h := range headerFlags {
+			ha := strings.Split(h, ": ")
+			if i := w.Header().Get(ha[0]); i != "" {
```

Comment:
I think that whether using a slice or a map, we could always use `w.Header().Add`. WDYT?

Reply:
Yes. Changed.